dali/server/daserver.cpp

```diff
@@ -139,6 +139,61 @@ void usage(void)
     printf("--daemon|-d <instanceName>\t: run daemon as instance\n");
 }
 
+static IPropertyTree *secMgrPluginPropTree(const IPropertyTree *configTree)
+{
+    Owned<IPropertyTree>    foundTree;
+
+#ifdef _CONTAINERIZED
+    IPropertyTree *jwtConfigPTree = configTree->queryPropTree("jwt");
+
+    if (jwtConfigPTree)
+    {
+        StringAttr configName(jwtConfigPTree->queryProp("@objname"));
+
+        VStringBuffer secMgrXML("<SecurityManager name='%s' instanceFactoryName='createInstance' libName='libjwtSecurity.so' type='JwtSecurityManager'/>", configName.str());
+        foundTree.set(createPTreeFromXMLString(secMgrXML));
```

Comment:
You are taking double ownership here.  In a successful scenario it's fine because you return ownership and the caller releases, but if there is an exception this will leak.  I think it's better to use setown() and then when you return do "return foundTree.getClear()".  That will pass the ownership to the caller, but if there is an exception it will release the object.

Reply:
Fixed.